openstack/compute/v2/servers/requests.go

```diff
@@ -701,3 +701,68 @@ func ListAddressesByNetwork(client *gophercloud.ServiceClient, id, network strin
 	}
 	return pagination.NewPager(client, listAddressesByNetworkURL(client, id, network), createPageFn)
 }
+
+type CreateServerImageOpts struct {
```

Comment:
s/CreateServerImageOpts/CreateImageOpts

Since we're in the `servers` package, we can drop the 'Server' part of the names. It'll also be more consistent with the rest of the package.